### DISCUSSION TITLE
Milestone names are not updated in issue view

### DISCUSSION THREAD

**Discussion by Vampire** [community/community#47666] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Milestone names are not updated in issue view if milestones are renamed.
I usually have a "Next Version" milestone where I put fixed issues in.
Then on release I rename the Milestone to the concrete version I release.
In the left view of an issue it keeps displaying the old name though.

If you for example go to https://github.com/Vampire/setup-wsl/issues/42 and search for "milestone",
you will find "Vampire added this to the Next Version milestone" which should be "Vampire added this to the v1.3.4 milestone"
and "Vampire modified the milestones: v1.3.4, v1.3.6, Next Version" which should be "Vampire modified the milestones: v1.3.4, v1.3.6, v2.0.0"

I know from GitHub support, that this is currently considered a non-modifiable history entry.
But I personally don't agree, and like to suggest to change this.
The history entry would still be valid when taking the renaming into account, as it is still that milestone it was added at that time.
From a user perspective it is just unclear to me which milestone it was added if the milestone name is not the one that is current.

**Comment by alexvgjm** [comment]
I have the same problem. I made a mistake when naming a milestone and after changing it, the change was not reflected in the history of an assigned issue.

https://github.com/alexvgjm/point-it-out/issues/30

The milestone was mistakenly called v1.2.0 when the issue was assigned. It was then renamed to v0.2.0.

I hope this is not intentional and will be fixed one day.

**Comment by g000sha256** [comment]
That also affects me.
For example, the label changes are applied in the current history.
